### ISSUE TITLE
Official PlatformIO IDE for Atom

### ISSUE THREAD

**Issue by ivankravets** [vigetlabs/platomformio#20]
I've released today official version of [PlatformIO IDE for Atom](http://platformio.org/#!/platformio-ide). It has built-in @PlatformIO CLI tool and doesn't require additional steps from user's side. Only "Install Package" & " Use It".

@efatsi, I want to say big THANK YOU :+1:  for your efforts and this package. I don't know what we should to do with this package. Do you think that we should support both packages?

Ping @RomFouq, @IoTAdri, @donnib, @sergeychernyshev, @gingebot, @reagent, @jeancdc 

![PlatformIO IDE for Atom](http://docs.platformio.org/en/latest/_images/ide-atom-platformio.png)

**Comment by jeancdc** [comment]
I tried it and it works well!

But the intelligent code completion does'nt work. Is there an option to enable?

**Comment by ivankravets** [comment]
@jeancdc Dod you install Clang? Please read instruction http://docs.platformio.org/en/latest/ide/atom.html#clang-for-intelligent-code-autocompletion

**Comment by jeancdc** [comment]
Yes, I installed Clang but I didn't have include the Arduino.h header file.

Now it works. Thank!
